cluster/gce/gci/configure-helper.sh

```diff
@@ -3280,6 +3280,8 @@ oom_score = -999
 # See: https://github.com/kubernetes/k8s.io/issues/3411
 [plugins."io.containerd.grpc.v1.cri".registry.mirrors."k8s.gcr.io"]
   endpoint = ["https://registry.k8s.io", "https://k8s.gcr.io",]
+[plugins."io.containerd.grpc.v1.cri".registry.mirrors."registry.k8s.io"]
+  endpoint = ["https://mirror.gcr.io"]
```

Comment:
Just for my curiosity - how does it work with the line above? We're first mirroring k8s.gcr.io to registry.k8s.io and then mirroring registry.k8s.io to mirror.gcr.io ?

Reply:
we don't really need the 3281/3282 at this point right? (if there are still lingering usages of k8s.gcr.io, we can fix them when it fails)

Reply:
> we don't really need the 3281/3282 at this point right? (if there are still lingering usages of k8s.gcr.io, we can fix them when it fails)

Yes, we can drop this, nothing should be pulling from k8s.gcr.io anymore.

Doesn't mirror.gcr.io only cache dockerhub?